### PR TITLE
Component/home page

### LINKED ISSUE #83: Sponsor

### LINKED ISSUE #102: HomePage
Create a Home page

Mobile design in [Figma](https://www.figma.com/file/gGwSe7ilZCUmAIaJyUc4iY/Bristol-Tech-Fair-Design-System?node-id=496%3A3731)
Tablet design in [Figma](https://www.figma.com/file/gGwSe7ilZCUmAIaJyUc4iY/Bristol-Tech-Fair-Design-System?node-id=496%3A3329)
Desktop design in [Figma](https://www.figma.com/file/gGwSe7ilZCUmAIaJyUc4iY/Bristol-Tech-Fair-Design-System?node-id=453%3A0)

### PR DESCRIPTION
**Pull request by r1903**
Closes #83,
Closes #102

This pull request has sponsor component and Homepage.

### CODE REVIEW

File: src/pages/HomePage/HomePage.styles.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+import styled from 'styled-components';
+import mediaQuery from '../../utils/mediaQuery';
+
+export const BannerWrapper = styled.div`
+  background: ;
+  height: 166px;
```
Comment (admtomas) [inline src/pages/HomePage/HomePage.styles.js]: missing value

File: src/components/SponsorCard/SponsorCard.test.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import SponsorCard from './SponsorCard';
+
+describe('SponsorCard', () => {
+  test('renders SponsorCard component', () => {
+    render(<SponsorCard value="Hello" />);
+
+    expect(screen.getByText('Hello'));
+  });
+});
```
Comment (admtomas) [inline src/components/SponsorCard/SponsorCard.test.jsx]: here we have error inside test suite. instead value we need props from Sponsor card and some test to mach those values, for ex if src exist or its name

File: src/pages/HomePage/HomePage.test.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+import HomePage from './HomePage';
+
+describe('HomePage', () => {
+  test('renders HomePage component', () => {
+    render(<HomePage value="Hello" />);
+
+    expect(screen.getByText('Hello'));
+  });
+});
```
Comment (admtomas) [inline src/pages/HomePage/HomePage.test.jsx]: we need test to match home page. its throwing errors. add props matching home page and create some test based on them

**Review by admtomas — CHANGES_REQUESTED** [review]
Entire project is good. Just please look at tests. Please change them and then we merge the Page. 
Run yarn test then click at letter 'p' and type name of component. Would be easier to read results and change the values. For testing you can use the same props you used for Storybook.

File: src/pages/HomePage/HomePage.styles.js (hunk 1 above)
Comment (r1903) [reply src/pages/HomePage/HomePage.styles.js]: Removed background style, as background color is common for all the pages it is better to add in main component.

File: src/pages/HomePage/HomePage.test.jsx (hunk 3 above)
Comment (r1903) [reply src/pages/HomePage/HomePage.test.jsx]: Added test cases.

**Review by admtomas — APPROVED** [review]

**Pull request merged**